packages/ui/core/Button/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {fireEvent} from '@testing-library/react-native';
+import renderer from 'react-test-renderer';
+import {StyledTheme} from '@animavita/theme';
+
+import {Mount, ThemeContext} from '../../../tests/helpers';
+import Button from '../Button';
+
+describe('Button', () => {
+  const cb = jest.fn();
```

Comment:
I think you need to add the below snippet here, not sure though.
```ts
beforeEach(jest.clearMocks)
```

Reply:
> Great job. Check comments please.

Ok, thank you!